depends/packages/rust.mk

```diff
@@ -10,10 +10,12 @@ $(package)_sha256_hash_freebsd=20d062493d01f1816014fe9dbe883bda06f1828a6ddbfb7ee
 
 # Mapping from GCC canonical hosts to Rust targets
 # If a mapping is not present, we assume they are identical
+$(package)_rust_target_x86_64-apple-darwin11=x86_64-apple-darwin
```

Comment:
It's this going to need to be continually updated for new macOS versions?

Reply:
We would add additional mappings to the same Rust target.